doc/developer/guide-testing.md

```diff
@@ -260,11 +262,70 @@ provide most of the coverage, but it's worth adding some (*very*) basic
 testdrive tests (e.g., `> SELECT DATE '1999-01-01'`) to ensure that our pgwire
 implementation is properly serializing dates.
 
-Like the [Unit Tests](#unitintegration-tests), many testdrive tests require
-Zookeeper, Kafka, and the Confluent Schema Registry.
+#### Usage via mzcompose
+
+The easiest way to run testdrive tests is via mzcompose.
+
+The mzcompose configuration will automatically set up all of testdrive's dependencies, including
+Zookeeper, Kafka, the Confluent Schema Registry, and mock versions of AWS S3 and Kinesis. From the
+`test/testdrive` directory:
+
+```
+./mzcompose down -v
+BUILD_MODE=debug TD_TEST=*.td ./mzcompose run local
+```
+
+The `down` subcommand is needed to destroy any existing materialize
+containers with possibly old binaries. If this is not run, then even
+though a new materialize binary is built, it will not be used if there's
+an existing container.
```

Comment:
I'm 99% sure that I fixed this bug. See https://github.com/MaterializeInc/materialize/commit/c59895102da3ea9510e40419a44b3ccb2d7ef8b6#diff-867be7cba8219319a498f47017a5d0116d2535ba2339d879a3e51d6b394f5418.

Reply:
innnteresting, okay I'll get rid of this.